tickets/src/components/content/CreateContent.js

```diff
@@ -0,0 +1,290 @@
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import Select from 'react-select'
+import Head from 'next/head'
+import Media from '../../theme/media'
+import Layout from '../interface/Layout'
+import GlobalErrorConsumer from '../interface/GlobalErrorConsumer'
+import DatePicker from '../interface/DatePicker'
+import BrowserOnly from '../helpers/BrowserOnly'
+import { pageTitle } from '../../constants'
+
+export class CreateContent extends Component {
+  constructor(props) {
+    super(props)
+    const { now, locks } = props
+
+    this.state = {
+      lock: locks[0] || '',
```

Comment:
We should probably show an error to the user when they do not have locks deployed?

Reply:
Or maybe just don't show the pulldown?

Reply:
We already have a link to create new locks

Reply:
They can work for now